### PR TITLE
Add FA playlist results to `New Featured Artist: Krimek` news article

### PR DESCRIPTION
**Pull request by pishifat**
and count correctly (The Dark was added to the listing at the last minute)

### CODE REVIEW

**Comment by Walavouchey** [comment]
wouldn't it make way more sense to put this alongside the next fa announcement?

**Comment by pishifat** [comment]
i'm thinking of putting it there too. this is for anyone who read the initial announcement and was like "ok i'll check back as the post said to do"

File: news/2023/2023-07-08-new-featured-artist-krimek.md [hunk 1]
```diff
@@ -58,7 +58,46 @@ For those out of the loop, playlist leaders earn prizes based on [this cumulativ
...
 
-We're currently reprocessing the last round of Featured Artist playlists after some recent changes to how scoring is handled on osu!(lazer). We expect this to take anywhere from 24–48 hours, so please check back then if you're interested in the results or updates on their progress!
+Scoring algorithms in osu!(lazer) were adjusted while this playlist was ongoing, so scores set using the previous algorithms have been reprocessed to match the current scoring formulas. In some cases, users couldn't improve their scores because plays set with the previous algorithms generally resulted in higher scores.
+
+Because of this, we've decided to reward users based on their **highest placement** in the playlist *before* reprocessing or the playlist *after* reprocessing.
+
+#### osu!
+
+**Before reprocessing:**
+
```
Comment (Walavouchey) [inline news/2023/2023-07-08-new-featured-artist-krimek.md:68]: seems like you've combined tables

```suggestion
```

File: news/2023/2023-07-08-new-featured-artist-krimek.md [hunk 2]
```diff
@@ -58,7 +58,46 @@ For those out of the loop, playlist leaders earn prizes based on [this cumulativ
...
+| Playlist rank (before reprocess) | (after reprocess) | User | Points earned | Total points |
+| --: | :-- | :-- | :-- | :-- |
```
Comment (Walavouchey) [inline news/2023/2023-07-08-new-featured-artist-krimek.md:70]: looks like crap

image

```suggestion
| Rank (before) | (after) | User | Points earned | Total points |
| --: | --: | :-- | :-- | :-- |
```

image

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2023/2023-07-08-new-featured-artist-krimek.md [hunk 3]
```diff
@@ -58,7 +58,46 @@ For those out of the loop, playlist leaders earn prizes based on [this cumulativ
...
+| #8 | N/A | ::{ flag=US }:: [Stoppedpuma](https://osu.ppy.sh/users/12654568) | 1 | 1 |
+| #9 | N/A | ::{ flag=US }:: [this_cord1985](https://osu.ppy.sh/users/19123572) | 1 | 1 |
+| #10 | #7 | ::{ flag=US }:: [Topical Kek](https://osu.ppy.sh/users/14480574) | 1 | 1 |
+| N/A | #9 | ::{ flag=RU }:: [Daycore](https://osu.ppy.sh/users/5596337) | 1 | 1 |
+| N/A | #10 | ::{ flag=US }:: [Majernja](https://osu.ppy.sh/users/11379588) | 1 | 1 |
+
+#### osu!mania
+
+| Playlist rank (before reprocess) | (after reprocess) | User | Points earned | Total points |
+| --: | :-- | :-- | :-- | :-- |
```
Comment (Walavouchey) [inline news/2023/2023-07-08-new-featured-artist-krimek.md:87]:
```suggestion
| Rank (before) | (after) | User | Points earned | Total points |
| --: | --: | :-- | :-- | :-- |
```

**Review by peppy — APPROVED** [review]

**Pull request merged**